packages/components/src/styles/copyright/copyright.css

```diff
@@ -6,4 +6,14 @@
     content: var(--copyright);
     text-shadow: 0 1px 3px #515151bf;
   }
+
+  &--variant {
+    &-vertical {
+      &::after {
+        @apply pl-2;
+        writing-mode: vertical-rl;
```

Comment:
Using writing-mode here feels a bit and leads to unexpected errors:

![CleanShot 2024-09-19 at 15 13 27@2x](https://github.com/user-attachments/assets/92579e22-157e-4522-8fe3-3a53880195d0)

The copyright sign moves strange. I would vote more for

```css
/* Changes for vertical */
.sd-copyright--vertical::after{
  transform: rotate(270deg);
  transform-origin: top left;
  padding-bottom: 0;
  padding-left: 0;
  padding-top: var(--sd-spacing-2,.5rem);
}
```

etc.

Here's a codepen:

https://codepen.io/mariohamann/pen/dyBxJVN?editors=1110

Reply:
Okay, but if you reduce the screen size, the copyright text in your example won't break. With vertical-rl, it does.
Changed some styles, maybe you could check it again